test/wrappers/vroom_test.rb

```diff
@@ -756,4 +756,17 @@ def test_mixed_shipments_and_services
     assert_equal 0, result[:unassigned].size
     assert_equal 5, result[:routes][0][:activities].size
   end
+
+  def test_correct_route_collection
+    problem = VRP.lat_lon_two_vehicles
+    problem[:services].each{ |service|
+      service[:skills] = ['s1']
+    }
+    problem[:vehicles].last[:skills] = [['s1']]
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:vroom] }}, TestHelper.create(problem), nil)
+    assert result
```

Comment:
I think we can stop checking this. If we have no result then next line will fail and while debugging it will be easy to see result is missing.